src/backend/TrafficCourts/Citizen.Service/Models/Disputes/Dispute.cs

```diff
@@ -16,7 +16,7 @@ public class Dispute : DisputantContactInformation
     /// </summary>
     [JsonPropertyName("ticket_number")]
     [MaxLength(12)]
-    public string? TicketNumber { get; set; } = null!;
+    public string TicketNumber { get; set; } = string.Empty;
```

Comment:
Is required empty string better than optional null?

Reply:
It's better for this specific property because TicketNumber is not a nullable property in the database so I think it's better to assign empty string in case no value provided to avoid null pointer exception type of error.